### ISSUE TITLE
[Feature][Zeta][Shell Script] The whole cluster can be started by script

### ISSUE THREAD

**Issue by liugddx** [apache/seatunnel#4166]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

1. You can start the entire cluster with --all, like  `./bin/seatunnel-cluster.sh  --all `
2. You can stop the entire cluster with --all, like  `./bin/stop-seatunnel-cluster.sh  --all `
3. If without --all, a single node is still started/stoped by default.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by MonsterChenzhuo** [comment]
please assign to me; thx。

**Comment by MonsterChenzhuo** [comment]
seatunnel-cluster.sh、stop-seatunnel-cluster.sh Merge into one script,
add：
<img width="192" alt="图片" src="https://user-images.githubusercontent.com/60029759/224083120-ec53a33d-8253-4b43-b394-f44b485bdff9.png">
sh seatunnel-cluster.sh start
sh seatunnel-cluster.sh stop

The final effect：
<img width="320" alt="图片" src="https://user-images.githubusercontent.com/60029759/224083438-051e4831-bce0-4a6a-8918-822fe31c8af7.png">


cc @liugddx @TyrantLucifer  @EricJoy2048

**Comment by liugddx** [comment]
> 
I agree with you. By the way, now the parameters will be parsed in two places, one in the script and one in the code, and we need to agree that the parameters will not conflict.
